src/main/java/org/mastodon/mamut/io/exporter/labelimage/ExportLabelImageController.java

```diff
@@ -292,4 +287,32 @@ private static MamutFeatureComputerService getMamutFeatureComputerService( final
 		featureComputerService.setModel( model );
 		return featureComputerService;
 	}
+
+	/**
+	 * Calculate the number of frames to use for the export depending on the number of timepoints and the frame rate reduction.
+	 * <br>
+	 * Some examples:
+	 * <br>
+	 * <ul>
+	 *     <li>timepoints: 1,  frame rate reduction:  1 ->  1 frame</li>
+	 *     <li>timepoints: 1,  frame rate reduction: 10 ->  1 frame</li>
+	 *     <li>timepoints: 10, frame rate reduction: 10 ->  1 frame</li>
+	 *     <li>timepoints: 10, frame rate reduction:  1 -> 10 frames</li>
+	 *     <li>timepoints: 11, frame rate reduction: 10 ->  2 frames</li>
+	 *     <li>timepoints: 20, frame rate reduction: 10 ->  2 frames</li>
+	 * </ul>
+	 *
+	 * @param numTimePoints the number of timepoints
+	 * @param frameRateReduction the frame rate reduction
+	 * @return the resulting number of frames
+	 */
+	static int getResultingNumberOfFrames( final int numTimePoints, final int frameRateReduction )
+	{
+		int frames = numTimePoints / frameRateReduction;
+		if ( numTimePoints > frameRateReduction && numTimePoints % frameRateReduction != 0 && frameRateReduction > 1 )
+			frames++;
+		if ( frames == 0 )
+			frames = 1;
+		return frames;
```

Comment:
I think what this code tries to achieve is dividing two integers and rounding up.  To my knowledge the standard way of implementing this is as follows:
```java
int frames = divideAndRoundUp(numTimePoints, frameRateReduction);

// with:
private static int divideAndRoundUp(int numerator, int denominator) {
        return (numerator + denominator - 1) / denominator;
}
````
I would be more confident that the code is correct. If you change it accordingly.

Reply:
Good idea. Changed it.